src/test/clojure/xtdb/expression_test.clj

```diff
@@ -1388,16 +1387,21 @@
              (run-test 'log (double 2.0) (bigdec 8.0))))
 
     ;; least + greatest
-    (t/is (= {:res 2.0, :res-type [:union #{:f64 :decimal}]}
+
+    (t/is (= {:res 2.0, :res-type [:union #{:f64 [:decimal 32 1 128]}]}
              (run-test 'least (bigdec 3.0) (double 2.0))))
-    (t/is (= {:res 3.0, :res-type [:union #{:f64 :decimal}]}
+    (t/is (= {:res 3.0, :res-type [:union #{:f64 [:decimal 32 1 128]}]}
              (run-test 'greatest (double 3.0) (bigdec 2.0))))
 
-
     ;; TODO decide on behaviour
     ;; out of fixed precision range
     (t/is (thrown? RuntimeException
-                   (run-test '+ (bigdec 1E+19M) (bigdec 1E+19M))))
+                   (run-test '+ (bigdec 1E+32M) (bigdec 1E+64M))))
+
+    ;; FIXME not sure why this doesn't throw
```

Comment:
fixme

Reply:
There is lots of stuff to address in the EE, I will fix these FIXME's when I am tackling https://github.com/xtdb/xtdb/issues/4331.